pkg/crds/operator/operator.tigera.io_installations.yaml

```diff
@@ -1483,6 +1483,17 @@ spec:
                           on interfaces that do not match the given regex.
                         type: string
                     type: object
+                      sysctl:
```

Comment:
You should run `make manifests`, that should update the installation CRD here and then you'll also get the enums that will cause apply time validation to be created.

Reply:
thanks